plugin/trino-hudi/src/test/resources/hudi-testing-data/stock_ticks_cow_clustered/README.md

```diff
@@ -0,0 +1,74 @@
+# Context
```

Comment:
Thanks for adding this README file! Is it possible to create the table with SQL? We usually use SQL when creating the resource likes `plugin/trino-delta-lake/src/test/resources`.

Reply:
Do you mean Spark SQL instead of this minimal Spark pipeline?

Reply:
Yes.

Reply:
Changed the `README.md` file accordingly. When the `schemacommit` test is ready, I will also add a `README.md` accordingly.

Reply:
Please note my last comment below regarding this.